assets/Icons/optimized/mail.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="320" height="320"><path d="M20.647 88.803C23.317 76.896 33.954 68 46.667 68h226.666c12.657 0 23.255 8.818 25.984 20.646l-139.47 70.977-139.2-70.82zM20 121.22l132.435 64.915c2.334 1.348 4.907 1.91 7.41 1.767 2.505.142 5.077-.42 7.41-1.767L300 121.042v120.29c0 14.73-11.94 26.668-26.667 26.668H46.667C31.94 268 20 256.06 20 241.333V121.22z"/></svg>
```

Comment:
missing viewBox

Reply:
oh duh this is in Assets nevermind

Reply:
Still it should be correct here. Basically we have replace all these and make sure they are right when we get the new ones.